envisage/plugins/ipython_kernel/internal_ipkernel.py

```diff
@@ -3,11 +3,18 @@
 https://github.com/ipython/ipython/blob/2.x/examples/Embedding/internal_ipkernel.py
 
 """
+from distutils.version import StrictVersion as Version
```

Comment:
`packaging.version.Version` is more up-to-date, but requires an external dependency (not a particularly onerous requirement, since `setuptools` already depends on `packaging`). This is probably good enough for this purpose.

Reply:
yes, I wanted to avoid adding any external dependencies